package.json

```diff
@@ -57,7 +57,7 @@
     "@angular/platform-browser": "~12.0.0",
     "@angular/platform-browser-dynamic": "~12.0.0",
     "@angular/router": "~12.0.0",
-    "@cds/core": "^5.0.0",
+    "@cds/core": ">=5.0.0 || <7.0.0",
```

Comment:
#147 closed in favour of this

Reply:
Wouldn't this match any version of @cds/core? Any version is going to be either greater than 5 *or* less than 7. Am I missing something?

Reply:
Also, this is only the devDependencies. Did you mean to update the peerDependencies in /projects/angular/package.json ?

Reply:
>Wouldn't this match any version of @cds/core? Any version is going to be either greater than 5 or less than 7. Am I missing something?

I think I left a comment in the original PR about this - currently, yes, it would match any version, but I wasn't sure if we wanted to exclude a hypothetical version 7 in the future since we won't know for sure. It's probably overkill though

Reply:
If that's the case shouldn't it be `&&` instead of `||`?

Reply:
Just read the [node-semver](https://github.com/npm/node-semver) spec and looks like it should be `>=5 <7`